### PR TITLE
Deprecate GetMaxClients

### PR DESCRIPTION
**Pull request by RumbleFrog**
🎉

### CODE REVIEW

File: plugins/include/clients.inc [hunk 1]
```diff
@@ -226,7 +226,7 @@ forward void OnClientPostAdminFilter(int client);
 forward void OnClientPostAdminCheck(int client);
 
 /**
- * This function will be deprecated in a future release.  Use the MaxClients variable instead.
+ * This function is deprecated. Use the MaxClients variable instead.
  *
  * Returns the maximum number of clients allowed on the server.  This may
```
Comment (Headline) [inline plugins/include/clients.inc]: You can remove this whole paragraph explaining it's use, along with the @return. It's only comment should be about it's deprecation.
```
 /**
   * This function is deprecated. Use the MaxClients variable instead.
   */
```

Comment (Headline) [reply plugins/include/clients.inc]: Actually, it looks like we don't do this anywhere else. It's fine as-is.

**Comment by 404UserNotFound** [comment]
Yay RumbleFrog

**Comment by RumbleFrog** [comment]
Done 🎉

File: plugins/include/clients.inc (hunk 1 above)
Comment (Headline) [reply plugins/include/clients.inc]: *Note*: Apparently we don't do this anywhere else, so feel free to ignore this comment. Thanks!

Comment (404UserNotFound) [reply plugins/include/clients.inc]: I've seen a few places where deprecation comment blocks exist. Let me take a quick gander and pull some examples up.

**Comment by asherkin** [comment]
Thanks!

**Pull request merged**